README.md

```diff
@@ -1,41 +1,43 @@
 # Terraforming OVH Public Cloud 
 
-This repo contains commons resources to interact with [OVH Public Cloud](https://ovhcloud.com/) using [Terraform](https://www.terraform.io/). 
+This repo contains commons resources to interact with [OVH Public Cloud](https://www.ovh.co.uk/public-cloud/instances/) using [Terraform](https://www.terraform.io/). 
 
-# Journey
+# Description
+
+We provide a step-by-step guide on how to use [Terraform](https://www.terraform.io/) with [OVH Public Cloud](https://www.ovh.co.uk/public-cloud/instances/).
 
-We provide a step-by-step guide on how to use [Terraform](https://www.terraform.io/) with [OVH Public Cloud](https://ovhcloud.com/).
+This is mainly for learning purpose but the use case presented can be use to deploy real infrastructures. Here, we'll deploy a web application on multiple nodes and multiple regions to be able to address important loads and provide redondency on two separated datacenters. We'll use the well known Wrodpress blog CMS to keep it simple.
```

Comment:
i think it is redundancy

Reply:
Yes it is, this is called "pedagogic" :p